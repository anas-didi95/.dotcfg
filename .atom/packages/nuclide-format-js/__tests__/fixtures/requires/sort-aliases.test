Aoel.Map();
Immutable.List();
Zena.Set();
