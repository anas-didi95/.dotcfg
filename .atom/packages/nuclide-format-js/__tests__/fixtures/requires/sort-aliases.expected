const Aoel = require('Aoel');
const Immutable = require('immutable');
const Zena = require('Zena');

Aoel.Map();
Immutable.List();
Zena.Set();
